lib/versions/version-info.js

```diff
@@ -220,6 +222,8 @@ if (NO_REMOTE_REQUESTS) {
   console.log(' - this is due to the "NG1_BUILD_NO_REMOTE_VERSION_REQUESTS" environment variable being defined.');
   console.log(' - be aware that the generated docs may not have valid or the most recent version information.');
   console.log('==============================================================================================');
-}
-console.log('CDN version:', cdnVersion.raw);
-console.log('Current version:', exports.currentVersion.raw);
\ No newline at end of file
+  console.log('Current local version', exports.currentVersion.raw);
+}  else {
+  console.log('CDN version:', cdnVersion.raw);
+  console.log('Current version:', exports.currentVersion.raw);
+}
```

Comment:
We still have a `cdnVersion` that is going to be used in the docs, so I think it is worth knowing what we are using.